tests/test_api_v1.py

```diff
@@ -189,6 +189,40 @@ def test_fetch_paginated_requests(
     assert type(fetched_requests[0]['dependencies']) == list
 
 
+def test_create_request_filter_state(app, auth_env, client, db):
+    """Test that requests can be filtered by state."""
+    repo_template = 'https://github.com/release-engineering/retrodep{}.git'
+    # flask_login.current_user is used in Request.from_json, which requires a request context
+    with app.test_request_context(environ_base=auth_env):
+        for i in range(10):
```

Comment:
Only two entries are necessary to test filtering. Each entry will just have a different state. By just having two, it makes the test a little faster since it reduces the amount of disk IO necessary.